### PR TITLE
Feature/e UI 8694

### PR DESCRIPTION
**Pull request by connorpgpmcelroy**
Resolves #https://tools.hmcts.net/jira/browse/EUI-8694 & https://tools.hmcts.net/jira/browse/EUI-8695

Notes:
* Adds display of registered and international address pages to register other org story
* Navigation also functional
* Validation will be present in other tickets but have placed some preliminary form groups on components

### CODE REVIEW

File: projects/exui-common-lib/src/lib/models/address.model.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// tslint:disable:variable-name
+export class AddressModel {
+  public addressLine1 = '';
```
Comment (johnbenjamin-hmcts) [inline projects/exui-common-lib/src/lib/models/address.model.ts:3]: The class should have members defined and not initialised?

File: projects/exui-common-lib/src/lib/components/write-address/write-address.component.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<div class="govuk-form-group">
+
+  <div *ngIf="!shouldShowDetailFields()">
+    <div class="govuk-form-group bottom-30 postcodeLookup" id="addressLookup"
```
Comment (johnbenjamin-hmcts) [inline projects/exui-common-lib/src/lib/components/write-address/write-address.component.html:4]: I can't find a styling for bottom-30. The GDS styling https://design-system.service.gov.uk/styles/spacing/ can be used instead.

File: projects/exui-common-lib/src/lib/components/write-address/write-address.component.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+      <select class="form-control govuk-select ccd-dropdown addressList" id="addressList" name="address"
+        [formControl]="addressList" (change)="addressSelected()" focusElement>
+        <option *ngFor="let addressOption of addressOptions" [ngValue]="addressOption.value">
+          {{addressOption.description}}
+        </option>
+      </select>
+    </div>
+
+    <a class="manual-link govuk-link bottom-30" *ngIf="!shouldShowDetailFields()" (click)="blankAddress()"
```
Comment (johnbenjamin-hmcts) [inline projects/exui-common-lib/src/lib/components/write-address/write-address.component.html:29]: Same as above.

**Review by johnbenjamin-hmcts — CHANGES_REQUESTED** [review]

File: projects/exui-common-lib/src/lib/models/address.model.ts (hunk 1 above)
Comment (connorpgpmcelroy) [reply projects/exui-common-lib/src/lib/models/address.model.ts:3]: this is consistent with the toolkit component but I will change

File: projects/exui-common-lib/src/lib/components/write-address/write-address.component.html (hunk 2 above)
Comment (connorpgpmcelroy) [reply projects/exui-common-lib/src/lib/components/write-address/write-address.component.html:4]: again, follows from toolkit component, will remove and see if I need to reset any spacing

File: projects/exui-common-lib/src/lib/components/write-address/write-address.component.html (hunk 3 above)
Comment (connorpgpmcelroy) [reply projects/exui-common-lib/src/lib/components/write-address/write-address.component.html:29]: same reply as above

**Pull request merged**